### PR TITLE
publish netclient docker workflow

### PR DESCRIPTION
**Pull request by mattkasun**

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Basic dependabot.yml file with minimum configuration for gomod.
+
+version: 2
+updates:
+  # Enable version updates for netmaker
+  - package-ecosystem: "gomod"
+    directory: "/"
+    # Check for updates every day (weekdays)
```
Comment (Aceix) [inline .github/dependabot.yml:8]: Guess the comment should be "Checks for updates once a week"

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    schedule:
+      interval: "weekly"
+    target-branch: "develop"
+  # Enable version updates for netclient
+  - package-ecosystem: "gomod"
+    directory: "/netclient"
+    # Check for updates every day (weekdays)
```
Comment (Aceix) [inline .github/dependabot.yml:15]: Same as above

File: .github/dependabot.yml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+updates:
+  # Enable version updates for netmaker
+  - package-ecosystem: "gomod"
+    directory: "/"
+    # Check for updates every day (weekdays)
+    schedule:
+      interval: "weekly"
+    target-branch: "develop"
+  # Enable version updates for netclient
+  - package-ecosystem: "gomod"
```
Comment (Aceix) [inline .github/dependabot.yml:13]: Will the bot raise a PR? Because I'm pretty sure we'll not want to jump on every latest dependency version

**Review by Aceix — CHANGES_REQUESTED** [review]

File: .github/dependabot.yml (hunk 3 above)
Comment (mattkasun) [reply .github/dependabot.yml:13]: yes, look at the PR in netmaker repo 1817, 1818, 1819 for example

**Review by Aceix — APPROVED** [review]

**Pull request merged**